### ISSUE TITLE
Inaccurate NPC dialogue

### ISSUE THREAD

**Issue by Spodey** [edisleado/Hitman-Bug-Tracker#14]
Should an accident occur, an NPC may say 'call 911' despite the level being in France/Italy/etc.

**Comment by edisleado** [comment]
Nice catch, but I'm not entirely sure if this counts as a 'bug' per se. I've labelled this as an enhancement.

Kind of weird that they'd even have '911' in the script considering the team is mainly Danish or English? Heh.

**Comment by Spodey** [comment]
Yeah, it isn't really a bug but I thought this was the best place for it to go.

I also see that you changed the titles, I'll be more precise next time :D
